### PR TITLE
Fix clear and undo controls

### PR DESCRIPTION
**Pull request by codebarista**
The Clear and Undo controls were not working. Here is a possible fix. Note: I am not that deep into frontend.

### CODE REVIEW

File: resources/js/controls/Undo.js [hunk 1]
```diff
@@ -38,8 +38,9 @@ class Undo extends Control {
         const map = this.getMap();
 
         map.getInteractions().getArray().forEach((obj) => {
-            if (obj.constructor.name === 'Draw2') {
```
Comment (mostafaznv) [inline resources/js/controls/Undo.js:41]: This line is correct; however, we should change 'Draw2' to 'Draw'

File: resources/js/controls/Undo.js [hunk 2]
```diff
@@ -38,8 +38,9 @@ class Undo extends Control {
...
+            try {
                 obj.removeLastPoint()
```
Comment (mostafaznv) [inline resources/js/controls/Undo.js:42]: However, this line does not work as expected.
I opened an issue on their GitHub. We should wait for their reply.

https://github.com/openlayers/openlayers/issues/16150

So, I think you should revert your changes on this part, except the `Draw2` that must change to `Draw`

**Comment by mostafaznv** [comment]
Hi @codebarista 
Thank you for your pull request, I appreciate it.

Your bug fix for the `Clear` button works perfectly. However, the one for the `Undo` button doesn't work properly.
I have added some comments to your code. Please review and address them, and once done, I can merge your pull request.

File: resources/js/controls/Undo.js (hunk 2 above)
Comment (mostafaznv) [reply resources/js/controls/Undo.js:42]: Based on the response to the attached issue, I believe that if you rename `Draw2` to `Draw`, the `Undo` button will function correctly.

**Comment by codebarista** [comment]
Hi @mostafaznv, 

Thanks for review. The try-catch was actually an ugly quick fix. The problem in the prod build was the minifing of the constructor name. It is only "Draw" in the watcher.

After digging deeper, I found the best solution by using the instance of the obj:

https://dev.to/gaurangdhorda/constructor-name-is-not-working-in-production-3ik4#comment-13ham

That should always work:

```
if (obj instanceof Draw) {
    obj.removeLastPoint()
}
```

**Comment by mostafaznv** [comment]
Your PR has been merged, and your changes have been released under v[4.0.3](https://github.com/mostafaznv/nova-map-field/releases/tag/4.0.3)

Thank you very much for your contribution.

**Comment by codebarista** [comment]
The update is on it's way to production. Thanks for coding.

**Pull request merged**